docs/General/release-process.md

```diff
@@ -0,0 +1,111 @@
+     1. Enter the release version as "Choose a tag", and select to
+        create it on publish.
+     2. Select `main` as target.
+     3. Click "Generate release notes" to automatically fill out title
+        and description.
+     4. Ensure that "Set as the latest release" is selected before
+        publishing the release.
+
+
+# Update DPL React for local development
+
+These steps are not critical for release, but are required for
+developers to get the latest version of the design system when
+developing.
+
+You might not have the required permissions for this step, in that
+case, continue to releasing DPL React and return to this later.
+
+https://github.com/danskernesdigitalebibliotek/dpl-react
+
+1. Create a new branch locally from `develop` (remember to `git pull` first).
+2. Ensure you're using the right version of `node`. You can use `nvm
+   use` for this.
+3. Update the version of the design system used by running `yarn add
+   @danskernesdigitalebibliotek/dpl-design-system@[version]`, you can
+   find the latest version of the design system [in the package on
+   Github](https://github.com/danskernesdigitalebibliotek/dpl-design-system/pkgs/npm/dpl-design-system)
+   NOTE: You need to be certified for this.
+4. Commit and push the changes.
+5. Create a pull request from your branch against `develop`.
+6. Wait for actions to turn green.
+7. Merge the pull request.
+
+# DPL React
+
+https://github.com/danskernesdigitalebibliotek/dpl-react
+
+1. Create a pull request from the `develop` against `main`.
+2. Wait for actions to go green.
+3. Merge pull request.
+4. [Create a new
+   release](https://github.com/danskernesdigitalebibliotek/dpl-react/releases/new)
+   in the same way as with the Design system.
+
+# DPL CMS
+
+1. Create a new branch locally from `develop` (remember to `git pull` first).
+2. Merge `main` into your branch.
+3. Update the version of the Design System used: `RELEASE=[version]
+   VERSION=[version] task dev:composer:update-design-system`. Refer to
+   `task dev:composer:update-design-system --summary` if needed.
+4. Update the version of React components used: `RELEASE=[version]
+   VERSION=[version] task dev:composer:update-react`
+5. Commit, push your branch and create a pull request against `develop`.
+6. Wait for actions to go green.
+7. Merge your pull request.
+8. Create a new pull request from `develop` against `main`.
+9. Wait for actions to go green.
+10. Merge your pull request.
+11. [Create a new
+    release](https://github.com/danskernesdigitalebibliotek/dpl-cms/releases/new)
+    in the same way as with the Design system.
+
+# Deployment to staging
+
+In order to deploy the new release to staging so that DDF can test it,
+refer to [Platform
+runbook](../../DPL-Platform/runbooks/how-release-a-new-version-for-approval-testing/).
+This will require access to Azure in order to run the deployment.
+
+# JIRA and communication
```

Comment:
```suggestion
## JIRA and communication
```

Reply:
Don't agree with the last two, deployment and kommunication isn't part of DPL CMS.

Maybe there should be two H1s: "Creating the release" and "Deployment and announcement", where we move the current headlines in under.